CMakeLists.txt

```diff
@@ -68,7 +68,7 @@ set(GLIB_MINIMUM 2.36)
 set (SERVICE_DEPS libayatana-common>=0.9.3 gio-unix-2.0>=${GLIB_MINIMUM} glib-2.0>=${GLIB_MINIMUM} properties-cpp>=0.0.1)
 
 if (ENABLE_COLOR_TEMP)
-    list (APPEND SERVICE_DEPS libgeoclue-2.0 accountsservice)
+    list (APPEND SERVICE_DEPS libgeoclue-2.0 accountsservice rda x11 xrandr)
```

Comment:
It would be good to have the rda feature optional (as build time switch), because in UT we don't want to ship that library. Like with all other indicators. (Not continuing with the review until this is resolved).

Reply:
Done.